### ISSUE TITLE
A few random errors from ESLint

### ISSUE THREAD

**Issue by pdehaan** [n1k0/readinglist-client#28]
I randomly ran ESLint against the **readinglist-client** repo for giggles, and it spit out the following errors/warnings:

```
$ eslint . | grep -v "Unexpected reserved word"

gulpfile.js
    6:25  error  Strings must use doublequote                                    quotes
    7:21  error  Strings must use doublequote                                    quotes
    8:23  error  Strings must use doublequote                                    quotes
    9:21  error  Strings must use doublequote                                    quotes
   10:21  error  Strings must use doublequote                                    quotes
   11:21  error  Strings must use doublequote                                    quotes
   13:20  error  Strings must use doublequote                                    quotes
   26:32  error  Extra space before value for key "NODE_ENV"                     key-spacing
   27:32  error  Extra space before value for key "MAX_ITEMS_PER_PAGE"           key-spacing
   28:70  error  Trailing comma                                                  no-comma-dangle
   51:19  error  Use path.join() or path.resolve() instead of + to create paths  no-path-concat
  121:27  error  done is defined but never used                                  no-unused-vars
  130:4   error  Don't use process.exit(); throw an error instead                no-process-exit
  160:6   error  Infix operators must be spaced                                  space-infix-ops

src/test/test_test.js
  5:41  error  reject is defined but never used  no-unused-vars

...

✖ 45 problems
```

And then all these are errors about "error  Unexpected reserved word" where the reserved word is `import`, I believe...

```
src/js/api.js
src/js/batch.js
src/js/components/Alert.js
src/js/components/App.js
src/js/components/ArticleEntry.js
src/js/components/ArticleForm.js
src/js/components/ArticleList.js
src/js/components/Auth.js
src/js/components/Button.js
src/js/components/Icon.js
src/js/components/Panel.js
src/js/components/Viewer.js
src/js/flux.js
src/js/main.js
src/test/api_test.js
src/test/batch_test.js
src/test/components/App_test.js
src/test/components/ArticleEntry_test.js
src/test/components/ArticleForm_test.js
src/test/components/ArticleList_test.js
src/test/components/Auth_test.js
src/test/components/Button_test.js
src/test/components/Panel_test.js
src/test/components/Viewer_test.js
src/test/components/index.js
src/test/flux/ArticleStore_test.js
src/test/flux/AuthStore_test.js
src/test/flux/index.js
src/test/index.js
src/test/utils.js
```

**Comment by n1k0** [comment]
Fixed by 9970cd74018cbfad1074b0dc23174638322ed5bb

**Comment by n1k0** [comment]
Please note that the project has now moved to eslint through #38.
